README.md

```diff
@@ -101,14 +132,102 @@ credentials, ie:
 }
 ```
 
-## Publishing
+The contents of this file can be found in lastpass with the name
```

Comment:
I disagree that it's a yak shave, and when it needs to be done, it would save that user a shitload of time hunting around a portal UI they don't know how to navigate if we just gave them a little help now.